### ISSUE TITLE
Features. Shorten some links spanning 2 lines

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#2115]
## Description of issue

windows 11
voice access
edge

[Fable Video](https://app.makeitfable.com/share/685ec98a816251063689d19d?video=1 ) at 9:11

Describe what is failing
- There is a long link spanning 2 lines. Not an issue, but shorter link could help readability.

## Potential fix

- What change could fix this issue?
- What nuggets of information could help us fix the issue

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

## Acceptance Criteria

- [ ] Should this issue be added to the A11y Statement if it cannot be resolved soon?

**Comment by jzbahrai** [comment]
Similar to https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/gh/cds-snc/notification-planning/2113, marking this as design for information on the line/ if they need us to look at it.

**Comment by schancase** [comment]
Just updated this to make it a task (vs issue with no type)
